iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -469,43 +488,25 @@ - (void)setStarted:(BOOL)started
   }
 }
 
-- (BOOL)start
+- (void)requestPermissionIfNeededAndStartUpdatingLocationFor:(CLLocationManager *)manager
 {
-  MWMVoidBlock doStart = ^{
-    LOG(LINFO, ("startUpdatingLocation"));
-
-    CLLocationManager * locationManager = self.locationManager;
-    if ([locationManager respondsToSelector:@selector(requestWhenInUseAuthorization)])
-      [locationManager requestWhenInUseAuthorization];
-
-    [locationManager startUpdatingLocation];
-
-    setPermissionRequested();
-
-    if ([CLLocationManager headingAvailable])
-      [locationManager startUpdatingHeading];
-  };
+  LOG(LINFO, ("startUpdatingLocation"));
+  
+  if ([manager respondsToSelector:@selector(requestWhenInUseAuthorization)])
```

Comment:
This method is supported from iOS 8, right? Is this if needed?

Reply:
removed - this is really outdated check